ksqldb-functional-tests/src/test/resources/sql-tests/test.sql

```diff
@@ -0,0 +1,119 @@
+-- this test file is a "meta-test" that tests the basic functionality of the KsqlTester
+-- eventually, we plan to remove this test in favor of migrating the QTTs over to use this format
+-- directly
+
```

Comment:
I see anything related to windowing is missing. Is this because it is not supported in this PR yet?

Reply:
Windowed keys aren't yet supported in our grammar so it'll take a bit of extra work to add support for windowing - I'll work on this after I have some basic query upgrade tests written